src/data/polynomial/degree/definitions.lean

```diff
@@ -805,6 +805,13 @@ from calc degree (C a) ≤ 0 : degree_C_le
                    ... = degree X : degree_X.symm,
 by rw [degree_sub_eq_left_of_degree_lt this, degree_X]
 
+@[simp] lemma degree_X_add_C (a : R) : degree (X + C a) = 1 :=
```

Comment:
Can you use this lemma to golf the proof of the `X_sub_C` variant
(using `rw [sub_eq_add_neg, <- C_neg]`)?